scripts/lint

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh -e
 
 autoflake --in-place --recursive wrapwith tests
-isort --recursive wrapwith tests
+isort wrapwith tests
```

Comment:
Is recursive the default now or did you decided to remove? :)

Reply:
Yep it's the default now (sorry should have highlighted that) - see the [release notes for 5.0](https://github.com/PyCQA/isort/blob/main/CHANGELOG.md#500-penny---july-4-2020)

Reply:
Nice - no probs at all :)